deploy/00_Deploy_Mocks.js

```diff
@@ -1,27 +1,38 @@
-const DECIMALS = '18'
-const INITIAL_PRICE = '200000000000000000000'
-
 module.exports = async ({
     getNamedAccounts,
     deployments,
     getChainId
 }) => {
+    const DECIMALS = '18'
+    const INITIAL_PRICE = '200000000000000000000'
     const { deploy, log } = deployments
     const { deployer } = await getNamedAccounts()
-    let chainId = await getChainId()
-    let ethUsdPriceFeedAddress
+    const chainId = await getChainId()
     // If we are on a local development network, we need to deploy mocks!
     if (chainId == 31337) {
-        log("Local network deteteched! Deploying mocks...")
-        linkToken = await deploy('LinkToken', { from: deployer })
-        await deploy('MockV3Aggregator', { from: deployer, args: [DECIMALS, INITIAL_PRICE] })
-        await deploy('VRFCoordinatorMock', { from: deployer, args: [linkToken.address] })
-        await deploy('MockOracle', { from: deployer, args: [linkToken.address] })
+        log("Local network detected! Deploying mocks...")
+        const linkToken = await deploy('LinkToken', { from: deployer, log: true })
+        await deploy('EthUsdAggregator', {
+            contract: 'MockV3Aggregator',
+            from: deployer,
+            log: true,
+            args: [DECIMALS, INITIAL_PRICE]
+        })
+        await deploy('VRFCoordinatorMock', {
+            from: deployer,
+            log: true,
+            args: [linkToken.address]
+        })
+        await deploy('MockOracle', {
+            from: deployer,
+            log: true,
+            args: [linkToken.address]
+        })
         log("Mocks Deployed!")
         log("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!")
         log("You are deploying to a local network, you'll need a local network running to interact")
-        log("Please run `npx hardhat node` to interact with the deployed smart contracts!")
+        log("Please run `npx hardhat console` to interact with the deployed smart contracts!")
         log("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!")
     }
 }
-module.exports.tags = ['Mocks']
```

Comment:
I don't understand the different tags?

Reply:
Now you can specify which use-case you want to deploy:
- `all`: Deploy all contracts
- `feed`: Deploy mocks and PriceConsumerV3
- `api`: Deploy mocks and APIConsumer
- `vrf`: Deploy mocks and RandomNumberConsumer